lib/main.ts

```diff
@@ -273,13 +273,13 @@ export async function fetch<T = any>(
   if (!kinde.fetch) {
     throw new Error("fetch binding not available");
   }
-
+  console.log("here");
```

Comment:
@DanielRivers there are a few `console.log`s in this file, assume we don't want them